templates/emails/com/onboarding/welcome-proton-user.txt.jinja2

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+
+{% block content %}
```

Comment:
We need to create a logo for Proton and SL here, let me ping the designer :)

Reply:
:+1:

Reply:
I think we can use this logo:
![sl-provided-by-proton](https://user-images.githubusercontent.com/3167780/174448024-41820b93-e987-4b1b-bc64-5202ecc5b8ee.png)
![sl-provided-by-proton](https://user-images.githubusercontent.com/3167780/174448026-e675b69d-d291-49b3-ad4e-c480861c4ceb.svg)